File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int start=0;
        int temp=0;
        int end = numbers.length-1;
        while(start<end){
            temp=numbers[start]+numbers[end];
            if(temp==target){
                return new int[]{start+1,end+1};
            }
            if(temp<target){
                start++;
            }else{
                end--;
            }
        }
        return new int[]{-1,-1};
    }
}

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        i=0
        j=len(numbers)-1
        while i<j:
            res=numbers[i]+numbers[j]
            if res>target:
                j-=1
            elif res<target:
                i+=1
            else:
                return [i+1,j+1]
        